src/fios.cpp

```diff
@@ -205,12 +205,18 @@ const char *FiosBrowseTo(const FiosItem *item)
  */
 static void FiosMakeFilename(char *buf, const char *path, const char *name, const char *ext, const char *last)
 {
-	const char *period;
+	if (buf != nullptr) {
```

Comment:
This implies `buf` is permitted to be nullptr, but that would result in seprintf() later being called with a nullptr for buffer.

Reply:
Oops... There should be `path` instead of `buf`.
Will be fixed with next push.